src/neo/Persistence/MemoryStore.cs

```diff
@@ -31,31 +24,31 @@ public ISnapshot GetSnapshot()
             return new MemorySnapshot(innerData);
         }
 
-        public void Put(byte table, byte[] key, byte[] value)
+        public void Put(byte[] key, byte[] value)
```

Comment:
Tables (or magic prefixes) are still used by dBFT and Nep17Tracker plugins to store their data (see Nep17BalancePrefix/Nep17TransferSentPrefix/ConsensusStatePrefix). IIUC after this change whole DB is considered to contain only `ST_Storage` things, but these plugins will still store their data that could clash with contract storage. So it looks like we still need "tables".

Reply:
How it be excluded certain keys from the MPT, checking if it start from this keys?

Reply:
> Tables (or magic prefixes) are still used by dBFT and Nep17Tracker plugins to store their data

`Nep17Tracker` uses a separate database. The same goes for the dBFT plug-in now.

Reply:
> How it be excluded certain keys from the MPT, checking if it start from this keys?

Yes.

Reply:
> Nep17Tracker uses a separate database. The same goes for the dBFT plug-in now.

Never noticed that, thanks. We're actually using the same DB in neo-go.